app/controllers/appropriate_bodies/appropriate_bodies_controller.rb

```diff
@@ -26,6 +24,10 @@ def create
 
   private
 
+    def set_choose_organisation_form
```

Comment:
instead of using `before_action`, can we try set the attribute memoized?
```
def choose_organisation_form
    @choose_organisation_form ||= ChooseOrganisationForm.new(user: current_user)
end
```